### ISSUE TITLE
is this line necessary?

### ISSUE THREAD

**Issue by Tomdkim** [tensorlayer/srgan#147]
https://github.com/tensorlayer/srgan/blob/14184cb4db4d8b6079e45c2d6d55e5e6af07df7d/train.py#L93

Hi there, when referring to your code regarding computing perceptual loss, I was wondering if this line of code would be necessary. Since you are using pretrained vgg19 model, it seems like taking out "vgg.train()" and adding the parameter "is_train=False" within vgg() function seems like the right way. Let me know if I'm wrong about this :D

Otherwise, thanks for sharing your nice code!

**Comment by zsdonghao** [comment]
This one is necessary, the new API uses `model.train()` `model.eval()` for train/test switching.
